string value

string COLLISION = "collision"
string GOAL_REACHED= "goal_reached"
string GOAL_LOST= "goal_lost"
